### DISCUSSION TITLE
Tool for creating custom coco keypoint dataset

### DISCUSSION THREAD

**Discussion by chungusmungus555** [open-mmlab/mmpose#1979] [Q&A]
Hey all, I'm trying to generate a custom human pose dataset to finetune a pretrained model. Are there any tools that are really good but is also open-source?

**Comment by ly015** [comment]
[LabelBee](https://github.com/open-mmlab/labelbee-client) or [LabelMe](https://github.com/wkentaro/labelme) is probably what you need.

**Reply by chungusmungus555** [reply]
Hey, thanks for replying, so I would need to annotate both the bounding box and the keypoints for training, right?

**Reply by ly015** [reply]
Yes, usually both bbox and keypoint annotation are needed. You can also annotate only keypoints and use the minimum enclosing rectangle as a coarse approximation of the bounding box.

**Reply by chungusmungus555** [reply]
> Yes, usually both bbox and keypoint annotation are needed. You can also annotate only keypoints and use the minimum enclosing rectangle as a coarse approximation of the bounding box.

That will make annotation faster but is there a tradeoff in training?
